src/index.js

```diff
@@ -53,10 +53,11 @@ function stack(stack, idx) {
 
 function format(name, err, suite = '') {
 	let { details, operator='' } = err;
-	let idx = err.stack && err.stack.indexOf('\n');
-	if (err.name.startsWith('AssertionError') && !operator.includes('not')) details = compare(err.actual, err.expected); // TODO?
+	let idx = err.stack && err.stack.indexOf('\n');	
 	let str = '  ' + FAILURE + (suite ? kleur.red(SUITE(` ${suite} `)) : '') + ' ' + QUOTE + kleur.red().bold(name) + QUOTE;
 	str += '\n    ' + err.message + (operator ? kleur.italic().dim(`  (${operator})`) : '') + '\n';
+	
+	if (err.code === 'ERR_ASSERTION' && !operator.includes('not')) details = compare(err.actual, err.expects);
```

Comment:
```suggestion
	if (err.code === 'ERR_ASSERTION' && !operator.includes('not')) details = compare(err.actual, err.expected);
```

Reply:
There's no expected property on the class Assertion.

Reply:
![image](https://user-images.githubusercontent.com/47633543/186834125-543f3a91-22bf-4f2b-8fa7-f95f1c94aad7.png)